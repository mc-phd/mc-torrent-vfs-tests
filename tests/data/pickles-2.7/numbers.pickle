(dp1
S'creation date'
p2
I1449730287842
sS'info'
p3
(dp4
S'files'
p5
(lp6
(dp7
S'path'
p8
(lp9
S'1.txt'
p10
asS'length'
p11
I1
sa(dp12
S'path'
p13
(lp14
S'2.txt'
p15
asS'length'
p16
I2
sa(dp17
S'path'
p18
(lp19
S'3.txt'
p20
asS'length'
p21
I3
sasS'piece length'
p22
I16384
sS'name'
p23
S'numbers'
p24
sS'pieces'
p25
S"\x1ftd\x8eP\xa6\xa6p\x8e\xc5J\xb3'\xa1c\xd5Sk|\xed"
p26
ssS'encoding'
p27
S'UTF-8'
p28
s.