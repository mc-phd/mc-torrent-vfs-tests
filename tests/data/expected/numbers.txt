dr-xr-xr-x 1 user group 0 02-15-9999 17:44 .META
-r--r--r-- 1 user group 25 02-15-9999 17:44 .META/creation date
-r--r--r-- 1 user group 5 02-15-9999 17:44 .META/encoding
-r--r--r-- 1 user group 5 02-15-9999 17:44 .META/piece length
dr-xr-xr-x 1 user group 0 02-15-9999 17:44 numbers
-r--r--r-- 1 user group 1 02-15-9999 17:44 numbers/1.txt
-r--r--r-- 1 user group 2 02-15-9999 17:44 numbers/2.txt
-r--r--r-- 1 user group 3 02-15-9999 17:44 numbers/3.txt
